addons/payment_stripe/controllers/main.py

```diff
@@ -19,9 +18,45 @@
 
 class StripeController(http.Controller):
     _checkout_return_url = '/payment/stripe/checkout_return'
+    _onboarding_return_url = '/payment/stripe/onboarding/return/'
+    _onboarding_refresh_url = '/payment/stripe/onboarding/refresh/'
     _validation_return_url = '/payment/stripe/validation_return'
     WEBHOOK_AGE_TOLERANCE = 10*60  # seconds
 
+    @http.route(
+        _onboarding_return_url + '<int:acquirer_id>/<int:menu_id>', type='http', auth='public'
+    )
+    def stripe_onboarding_return(self, acquirer_id, menu_id):
+        """Stripe return URL
+
+            This route is used by Stripe to return after or during the onboarding.
+            Once called, the controller redirects to the payment acquirer.
+
+            :returns: Redirection to Stripe Acquirer Form
+        """
```

Comment:
```suggestion
        """ Redirect the user to the acquirer form of the onboarded Stripe account.

        This user is redirected to this route by Stripe after or during (if the user clicks on a
        dedicated button) the onboarding.
        
        :param int acquirer_id: The acquirer linked to the Stripe account being onboarded, as a
                                `payment.acquirer` id
        :param int menu_id: The menu from which the user started the onboarding step, as an
                            `ir.ui.menu` id
        """
```
Note that I dropped the `return` because I never found a convenient way to describe what `request.redirect` returns. Since the docstring title clearly states that the user is redirected, it's not a big deal.